docs/Modules/index.md

```diff
@@ -0,0 +1,83 @@
+# Modules
+
+## Base Classes
+
+Depending on which `ScriptModule` you desire the module to be in, you change the inheritance to allow for more events. All the possible events in the modules are documented within the the respective classes.
+
+| Script Module | CF Module        |
+| ------------- | ---------------- |
+| 1_Core        | `CF_ModuleCore`  |
+| 2_GameScript  | `CF_ModuleCore`  |
+| 3_Game        | `CF_ModuleGame`  |
+| 4_World       | `CF_ModuleWorld` |
+| 5_Mission     | `CF_ModuleWorld` |
```

Comment:
Twice World? No Mission module? I mean even if it did nothing you could still maybe make one and have it be a 1:1 inherit from World or does that not make much sense?

Reply:
The same base module class can be used for either scriptmodule.

Didn't want to add a new class when there was no real reason to due to the '#InitGlobals' issue.